static_src/main.js

```diff
@@ -56,9 +56,10 @@ function overview() {
 function org(orgGuid) {
   orgActions.toggleSpaceMenu(orgGuid);
   orgActions.fetch(orgGuid);
+  cfApi.fetchSpaces().then(() => spaceActions.fetchAllForOrg(orgGuid));
```

Comment:
This sticks out to me, since we fetch the org via action, but the spaces are fetched with the api directly. I'm not totally sure "what's right" but I feel like moving the action handling into cfApi gives the caller more control (because we have access to the request's promise).